src/sentry/api/endpoints/project_app_store_connect_credentials.py

```diff
@@ -212,7 +212,7 @@ def post(self, request: Request, project: Project) -> Response:
         config = serializer.validated_data
 
         config["type"] = "appStoreConnect"
-        config["id"] = uuid4().hex
+        config["id"] = uuid4()
```

Comment:
@flub can you remember a specific reason you went for `hex` ids?

Reply:
Because in the json we need this to be a string rather than a class instance.  i don't think you should be changing this should you?  what's the motivation to change this?

Reply:
Oh, so this does not automatically stringify the uuid? I just wanted all the IDs to have the same format. in this specific case to have hyphenated uuids.
But I will actually manually test this stuff tomorrow, I admit I haven’t done that ;-)
Although the tests have hyphenated UUIDs in them right now.